mg/cmd.go

```diff
@@ -22,6 +23,7 @@ type CmdOutputWriter struct {
 	closed bool
 }
 
+// Copy applies updaters to the object.
```

Comment:
should probably expand to make it clear that they're applied to a `new copy of the writer`, the current text implied that they're updating the existign object.